### PR TITLE
844 - analyst application view e2e screenshot test

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #844

### CODE REVIEW

File: db/data/e2e/001_application_received.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+begin;
+
+insert into ccbc_public.application_status
```
Comment (AntBush) [inline db/data/e2e/001_application_received.sql:3]: nit: can probably just use the 001_received_applications.sql for this

Comment (marcellmueller) [reply db/data/e2e/001_application_received.sql:3]: Oh I don't think that was there when I started this PR, I will update it thanks.

File: app/cypress/integration/analyst/application/[applicationId].spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+describe('The analyst application view', () => {
+  it('loads', () => {
+    cy.mockLogin('ccbc_analyst');
+    const mockedDateString = '2022-10-10';
+    const mockedDate = new Date(mockedDateString);
+    cy.useMockedTime(mockedDate);
+    cy.sqlFixture('e2e/reset_db');
+    cy.sqlFixture('e2e/001_intake');
+    cy.sqlFixture('e2e/001_application');
+    cy.sqlFixture('e2e/001_application_received');
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply app/cypress/integration/analyst/application/[applicationId].spec.js:10]: I considered that but I think I'll leave it basic for now as the only purpose of this test is to take a happo screenshot.

[the remark this answers is not in the document]
Comment (marcellmueller) [reply app/cypress/integration/analyst/application/[applicationId].spec.js:10]: hmmm no good shout I will update it so it stays more organized :+1:

**Pull request merged**